testing-framework/ec2/base/bootstrap_node_provisioner.tf

```diff
@@ -51,6 +51,7 @@ resource "null_resource" "setup-bootstrap-nodes" {
     inline = [
       "cd /home/${var.ssh_user}/subspace/",
       "git clone https://github.com/subspace/subspace.git",
+      "cd subspace",
```

Comment:
why `cd` into subspace when we are already in `subspace` directory?

Reply:
It's doing a cd into the cloned repo, the base subspace directory just to hold all the subspace related configurations and the repo is cloned inside there since it's building from the branch.